test/feature/configuration/serializer_lookup/using_serializable_resource_test.rb

```diff
@@ -0,0 +1,42 @@
+require 'test_helper'
+
+describe 'feature' do
+  describe 'configuration' do
+    describe 'serializer lookup' do
+      describe 'using SerializableResource' do
+        it 'uses the child serializer' do
```

Comment:
this'll be todo, as we'd just be testing different ways to configure -- which is open ended